share/etter.filter.examples

```diff
@@ -23,6 +23,14 @@ if (ip.proto == TCP) {
 }
 
 
+# log all IEC 61850 traffic when the destination ip is 192.168.1.2
+if (if (ip.dst == '192.168.1.2' && tcp.src == 102) {
```

Comment:
Have you tested this expression? I think this is not correct and would result in failure when compiling with `etterfilter`.

Reply:
It's a typo problem I wrote two if, my mistake.